Parcare/Services/ParkingLotService.cs

```diff
@@ -11,25 +11,29 @@ public class ParkingLotService
     {
         private List<ParkingLotModel> _parkedCarsList;
         private int _maxSlots;
+        private float _price;
+        private BankAccountService _bank;
 
-        public ParkingLotService(int totalSlots)
+        public ParkingLotService(int totalSlots, float price, BankAccountService bankAccounts)
         {
             _parkedCarsList = new List<ParkingLotModel>();
             _maxSlots = totalSlots;
+            _price = price;
+            _bank = bankAccounts;
         }
 
         public bool IsParkingPossible()
         {
             return (_maxSlots - _parkedCarsList.Count) > 0;
         }
-        public void ParkCar(string carNumber)
+        public void ParkCar(string carNumber, DateTime time)
         {
             if (IsParkingPossible())
```

Comment:
revert this if statement (like if it is not true return) and have the code inside again not nested